upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -300,6 +300,17 @@ func (c *ApplyClusterCmd) Run() error {
 		}
 	}
 
+	var nodeDockerConfig string
+	{
+		secret, _ := secretStore.Secret("nodedockercfg")
+		if secret != nil {
+			nodeDockerConfig, err = secret.AsString()
+			if err != nil {
+				return fmt.Errorf("error node docker config not a string? %q: %v", nodeDockerConfig, err)
```

Comment:
in a previous iteration I had it actually checking if the json is parsable, I think maybe I'll switch this back to that and maybe move it into the CLI code since that seems like a better place to provide the error.